index.js

```diff
@@ -179,6 +223,14 @@ const captureWebsite = async (url, options) => {
 		await page.click(options.clickElement);
 	}
 
+	if (options.scrollToElement) {
```

Comment:
I think this should be before the screenshot is captured, in case, for example, the `styles` option affects the scroll position.